### ISSUE TITLE
Twilio integration doesn't use MMS? Also, RC sent Emojis not rendered on cellphones

### ISSUE THREAD

**Issue by designgurudotorg** [RocketChat/Rocket.Chat#6745]
@sampaiodiego Any idea why emojis from iOS get rendered in Rocketchat when using Twilio/SMS in livechat but RocketChat-created emojis don't render on the receiver side? The receiving cellphone sees emojis from RC as :grin: for example (and not the emoji itself.)

*Also it looks like RC only sends/receives SMS - MMS messages sent to a Twilio/SMS/Rocketchat number show up as empty messages.

![screen shot 2017-04-20 at 11 38 50 am](https://cloud.githubusercontent.com/assets/12697545/25239439/f958ea06-25bd-11e7-9b6f-c5962ce35caa.png)

**Comment by sampaiodiego** [comment]
RC does not use UTF8 characters for emojis, that's why it's not rendered on phone.

**Comment by designgurudotorg** [comment]
AH! Okay, so the only way to do this properly then is to paste in the UTF emoji into RC chat - it seems to show up fine then on both sides (ref: http://unicode.org/emoji/charts/full-emoji-list.html)... though copying and pasting from elsewhere is a pain...

**Comment by designgurudotorg** [comment]
What about the MMS question @sampaiodiego ? The current RC-Twilio implementation is SMS only right? Any plans to expand that to optionally allow MMS?

**Comment by graywolf336** [comment]
@sampaiodiego can we use emojione to translate them into the UTF8 emojis? This could even be done on push notifications so emojis are rendered in push notifications. :)

**Comment by sampaiodiego** [comment]
yes @graywolf336 .. I tried already doing that for push notifications.. but it was more work to do than I thought initially :stuck_out_tongue:

**Comment by graywolf336** [comment]
Aww that's too bad :(

**Comment by engelgabriel** [comment]
Closing this PR in favor of https://github.com/RocketChat/Rocket.Chat/pull/39411, where we’re replacing Emojione with native Unicode emojis.
